blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -6,6 +6,8 @@
 import hudson.model.User;
 import hudson.tasks.Mailer;
 import hudson.tasks.UserAvatarResolver;
+import hudson.util.HttpResponses;
```

Comment:
`BlueUser` is a rest resource (extends `Resource`), so yes it exposes data as well as could expose other resources. This is fine we can add later, for this PR purpose its fine.